frontend/src/components/phones/dashboard/Dashboard.tsx

```diff
@@ -123,52 +123,35 @@ export const PhoneDashboard = () => {
   const phoneControls = (
     <div className={styles["phone-controls-container"]}>
       <div className={styles["phone-controls"]}>
-        <span
-          // This is a <span> rather than a <button> to allow the use of flexbox,
-          // but React Aria makes sure it's correctly recognised as a button
-          // thanks to `elementType` passed to `useToggleButton`:
+        <button
           {...forwardToggleButtonProps}
           ref={forwardToggleButtonRef}
-          className={`${styles["forwarding-toggle"]} ${
-            forwardToggleState.isSelected
-              ? styles["is-forwarding"]
-              : styles["is-blocking"]
-          }`}
+          className={styles["forwarding-toggle"]}
           title={
             forwardToggleState.isSelected
-              ? l10n.getString("phone-dashboard-forwarding-toggle-enabled")
-              : l10n.getString("phone-dashboard-forwarding-toggle-disabled")
+              ? l10n.getString(
+                  "phone-dashboard-forwarding-toggle-disable-tooltip"
+                )
+              : l10n.getString(
+                  "phone-dashboard-forwarding-toggle-enable-tooltip"
+                )
```

Comment:
Question: Is this the same logic but the strings are switched? This may need to be swapped?

Reply:
Ah heh, they're different stings, but apparently I used confusing string IDs :P The previous strings indicated what the current status is (enabled/disabled), whereas the new strings indicate what will happen if you click it (disable/enable, respectively). The new strings also explicitly have `-tooltip` in their IDs, so they are actually different strings.